content/en/docs/reference/kubectl/quick-reference.md

```diff
@@ -371,6 +371,8 @@ kubectl port-forward my-pod 5000:6000               # Listen on port 5000 on the
 kubectl exec my-pod -- ls /                         # Run command in existing pod (1 container case)
 kubectl exec --stdin --tty my-pod -- /bin/sh        # Interactive shell access to a running pod (1 container case)
 kubectl exec my-pod -c my-container -- ls /         # Run command in existing pod (multi-container case)
+kubectl debug my-pod -it --image=busybox            # Create an interactive debugging session witin existing pod and immediately attach to it
+kubectl debug node/my-node -it --image=busybox      # Create an interactive debugging session on a node and immediately attach to it
```

Comment:
Please use `busybox:1.28` per other examples, or bump all the examples in this page to a different version of BusyBox.